src/maybe.ts

```diff
@@ -1622,10 +1622,10 @@ export function last<T>(array: Array<T | null | undefined>): Maybe<T> {
   import Maybe from 'true-myth/maybe';
 
   let valid = [Maybe.just(2), Maybe.just('three')];
-  let allJust = Maybe.all(valid); // => Just([2, 'three']);
+  let allJust = Maybe.transposeArray(valid); // => Just([2, 'three']);
```

Comment:
all these examples should be `arrayTranspose`

Reply:
DOH. Thank you!